database/meta.go

```diff
@@ -128,8 +130,20 @@ func (m *MetaDBHandle) MarkDBAnalyzed(name string, complete bool) error {
 		return err
 	}
 
+	var versionTag string
+	if complete {
+		versionTag = m.res.Config.S.Version
+	} else {
+		versionTag = ""
+	}
```

Comment:
Same method handles setting the db analyzed flag true and false, dependent on the complete variable. if !complete, remove the tag